packages/components/src/select-control/types.ts

```diff
@@ -58,7 +58,7 @@ export interface SelectControlProps
 		 */
 		disabled?: boolean;
 	}[];
-	value?: string | string[];
+	value?: number | string | string[];
```

Comment:
The `<QueryControls>` component passes to the `<SelectControl>` a `value` [prop](https://github.com/WordPress/gutenberg/blob/921135f9ec7f1e54030672eb7d11ca28034bf15b/packages/components/src/select-control/index.tsx#L122) of type `number`.

<img width="1031" alt="Screen Shot 2022-12-21 at 4 59 48 PM" src="https://user-images.githubusercontent.com/4063887/209019824-47b47589-980f-4889-9db5-c0085f625695.png">

Reply:
And the React type [SelectHTMLAttributes ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/04d5ce209125122ab55ad7502c88a5ef05c1b9a8/types/react/index.d.ts#L2412) allows a `value` of `number`:

```ts
value?: string | ReadonlyArray<string> | number | undefined;
```